docs/user-guide/file-library/files.md

```diff
@@ -114,11 +114,7 @@ The file sidebar also includes the following details, which are not editable and
 
 ## Edit an Image
 
-Rotate, crop, flip, or adjust aspect ratios of an image.
-
-<video alt="Edit an Image" loop muted controls autoplay playsinline>
-  <source src="https://cdn.directus.io/docs/v9/app-guide/file-library/file-library-20220516A/edit-an-image-20220516A.mp4" type="video/mp4">
-</video>
```

Comment:
Are we completely removing the video on purpose? I think theres still value in it being there so users can get a feel for the UI even if it is slighty outdated with the focal point button missing.

Reply:
Yes, generally, I’m trying to remove them because I don’t think they ever should’ve been there. They have a really high maintenance burden, and I would rather replace them with screenshots if anything.

Reply:
I can't remember which site I saw this on, but they had official screenshots of features in their docs and presented them in slideshows. When you got to the end of the slides, there were multiple community generated videos of it, followed by a prompt to submit your own user generated video of this feature. They were surprisingly high quality videos, short, and some were in other languages with captions.

Reply:
Interesting. Maybe something we'll consider in the future, but for now most of the videos are wildly out of date and I'll be removing them as I spot them ahead of a proper sweep soon.